index.html

```diff
@@ -0,0 +1,303 @@
+          <div class="card-left">
+            <img src="images/facebook360.png" alt="Snapshot Protfolio" />
+          </div>
+          <div class="card-right">
+            <div class="newme-title-tonic primary-title">Facebook 360</div>
+            <div class="card-tonic3">
+              <h3 class="wordcanopi primary-text">Facebook</h3>
+              <h3><img src="./images/Counter.png" alt="counter count" class="dot" /></h3>
+              <h3 class="wordback third-text">Full Stack Dev</h3>
+              <h3><img src="./images/Counter.png" alt="counter count" class="dot" /></h3>
+              <h3 class="word2015 third-text">2015</h3>
+            </div>
+            <div class="description-tonic primary-text">
+              <p>
+                Exploring the future of media in Facebook's first Virtual Reality 
+                app; a place to discover and enjoy 360 photos and videos on 
+                Gear VR.
+              </p>
+            </div>
+            <li class="card-tonic5">
+              <h1 class="tonichtml primary-link">html</h1>
+              <h1 class="toniccss primary-link">Ruby on rails</h1>
+              <h1 class="tonicJava primary-link">CSS</h1>
+              <h1 class="tonicJava primary-link">JavaScript</h1>
+            </li>
+            <div class="buttontonic">
+              <button type="submit" class="go-buttontonic">See project</button>
+          </div>
+          </div>
+        </div>
+        <div class="card">
+          <div class="card-left">
+            <div class="newme-title-tonic primary-title">Uber Navigation</div>
+            <div class="card-tonic3">
+              <h3 class="wordcanopi primary-text">Uber</h3>
+              <h3><img src="./images/Counter.png" alt="counter count" class="dot" /></h3>
+              <h3 class="wordback third-text">Lead Developer</h3>
+              <h3><img src="./images/Counter.png" alt="counter count" class="dot" /></h3>
+              <h3 class="word2015 third-text">2018</h3>
+            </div>
+            <div class="description-tonic primary-text">
+              <p>
+                A smart assistant to make driving more safe, efficient, and fun 
+                by unlocking your most expensive computer: your car.
+              </p>
+            </div>
+            <li class="card-tonic5">
+              <h1 class="tonichtml primary-link">html</h1>
+              <h1 class="toniccss primary-link">Ruby on rails</h1>
+              <h1 class="tonicJava primary-link">CSS</h1>
+              <h1 class="tonicJava primary-link">JavaScript</h1>
+            </li>
+            <div class="buttontonic">
+              <button type="submit" class="go-buttontonic">See project</button>
+          </div> 
+          </div>
+          <div class="card-right">
+            <img src="images/ubernavi.png" alt="Snapshot Protfolio" />
+          </div>
+        </div>
+      </div>
+      <div class="firstcont">
+        <div class="card1">
```

Comment:
- [ ] I kindly suggest that you follow the Figma design closely in designing your about me section.
- [ ] Please kindly ensure that you space your social icons correctly.
- [ ] Please kindly ensure that you space your languages section correctly.
- [ ] Please kindly ensure that the about me takes 40%, while the professional skills on the other side takes 60% of the page width.
- [ ] Please kindly space your Javascript, HTML and CSS tags correctly as seen on the Figma design.

|   |  |
| ------------- | ------------- |
| Your Design  | <img width="898" alt="image" src="https://github.com/jovasal/complete-portfolio/assets/52934128/a635207f-ade7-40eb-8756-577190fb8adf">  |
| Figma Design  | 
<img width="548" alt="image" src="https://github.com/jovasal/complete-portfolio/assets/52934128/49ec7195-c235-43e9-a86a-4001b5c08399">  |